docs/concepts/CLQL.md

```diff
@@ -25,7 +25,7 @@ We plan to extend the Lexicons libraries to include:
 
 ## Lexicon SDK
 
-If you are interested in writing your own custom Lexicons please see the **[Lexicon SDK](https://github.com/codelingo/lexiconsdk)**.
+If you are interested in writing your own custom Lexicons, please reach out via  **hello@codelingo.io**.
```

Comment:
check who has access to this email (if anyone), even better - get them to join the slack chan!